theano/sandbox/gpuarray/comp.py

```diff
@@ -8,9 +8,17 @@
 # This is a big hack to avoid creating a second context on the card.
 from theano.sandbox.cuda.nvcc_compiler import (NVCC_compiler as NVCC_base,
                                                hash_from_file)
+
+from .type import get_context
+
 class NVCC_compiler(NVCC_base):
-    @staticmethod
-    def compile_args():
+    # This is for gof/cc.py
```

Comment:
Why this is needed? I check and with the curren Theano master, 
```
>>> theano.sandbox.gpuarray.comp.NVCC_compiler.__name__
'NVCC_compiler'
```

So it seam useless.